xarray/tests/test_units.py

```diff
@@ -1245,6 +1263,146 @@ def test_dot_dataarray(dtype):
     assert_equal_with_units(expected, actual)
 
 
+def delete_attrs(*to_delete):
+    def wrapper(cls):
+        for item in to_delete:
+            setattr(cls, item, None)
+
+        return cls
+
+    return wrapper
+
+
+@delete_attrs(
+    "test_getitem_with_mask",
+    "test_getitem_with_mask_nd_indexer",
```

Comment:
are there better ways to exclude inherited tests? I'd like to avoid marking them as skipped since they will probably never be passing.